src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -48,6 +48,7 @@ def _create_tasks(
         crash_report_timeout: Optional[int] = None,
         debug: Optional[List[TaskDebugFlag]] = None,
         ensemble_sync_delay: Optional[int] = None,
+        merge: bool = False,
```

Comment:
The changes in #199 don't support a mechanism to make tasks optional on the command line and will be a significantly difficult to implement at this stage.

Can we not include this addition here?  When we move to declarative templates, it would be useful to not remove functionality.